src/lib/models/licence-agreement.js

```diff
@@ -34,10 +45,12 @@ class LicenceAgreement extends Model {
     this._agreement = agreement;
   }
 
-  get dateSigned () { return this._dateSigned; }
+  get dateSigned () {
+    return this._dateSigned ? this._dateSigned.format(DATE_FORMAT) : this._dateSigned;
```

Comment:
I think if _dateSigned is null or undefined, this will return null or undefined. Is that intentinonal?

If this is intentional: Ignore me.

If this isn't intentional: I would suggest changing the `else` value to some sort of default fallback. E.g. Today's date or something.

Reply:
This is intentional